addons/de/xrechnung/extensions.go

```diff
@@ -0,0 +1,84 @@
+package xrechnung
+
+import (
+	"github.com/invopop/gobl/cbc"
+	"github.com/invopop/gobl/i18n"
+)
+
+// ExtKeyTaxRate is the key for the tax rate extension in XRechnung
+var ExtKeyTaxRate cbc.Key = "de-xrechnung-tax-rate"
```

Comment:
Is "tax rate" the correct name? The definitions also cover tax categories. I'd strongly suggest adding a comment with the link to the source information to be able to double check.